File: message-processor-reverse/src/main/java/message/processor/reverse/main/MessageProcessorApplication.java
package message.processor.reverse.main;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan("message.processor.reverse")
public class MessageProcessorApplication {

    public static void main(String[] args) {
        new SpringApplicationBuilder(MessageProcessorApplication.class).run(args);
    }

}

File: message-processor-joke/src/main/kotlin/message/processor/joke/main/MessageProcessorApplication.kt
package message.processor.joke.main

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@ComponentScan("message.processor.joke")
@EnableAutoConfiguration
@EnableConfigurationProperties
class MessageProcessorApplication

fun main(args: Array<String>) {
    SpringApplication.run(MessageProcessorApplication::class.java, *args)
}